lib/processors/minifier.js

```diff
@@ -34,12 +34,16 @@ const debugFileRegex = /((?:\.view|\.fragment|\.controller|\.designtime|\.suppor
  * @alias module:@ui5/builder.processors.minifier
  * @param {object} parameters Parameters
  * @param {module:@ui5/fs.Resource[]} parameters.resources List of resources to be processed
- * @param {boolean} [parameters.addSourceMappingUrl=true]
+ * @param {object} [parameters.options] Options
+ * @param {boolean} [parameters.options.addSourceMappingUrl=true]
  * 				Whether to add a sourceMappingURL reference to the end of the minified resource
  * @returns {Promise<module:@ui5/builder.processors.MinifierResult[]>}
  * 				Promise resolving with object of resource, dbgResource and sourceMap
  */
-module.exports = async function({resources, addSourceMappingUrl = true}) {
+module.exports = async function({resources, options: {addSourceMappingUrl} = {}}) {
+	if (addSourceMappingUrl === undefined) {
```

Comment:
is it possible to specify the default value also for nested parameters in the function declaration?

Reply:
Thanks. I thought that this wouldn't work when no options are provided. But it seems to work.

Reply:
Good point, thanks. I thought that it isn't possible when the options should be optional, but it seems to work.

Reply:
Fixed by https://github.com/SAP/ui5-builder/pull/709/commits/267fafc0da8ed36f859423cb1e2010b84c9fb11a